Tools/Benchmarks/STREAM/README.md

```diff
@@ -0,0 +1,131 @@
+# STREAM Sample
+
+This sample contains a [STREAM](http://www.cs.virginia.edu/stream/) implementation using DPC++ for CPU and GPU. 
+
+
+| Optimized for                       | Description
+|:---                               |:---
+| OS                                | Linux* Ubuntu* 20.04
+| Hardware                          | GEN9, Iris-Xe Max 
+| Software                          | Intel&reg; oneAPI DPC++ Compiler
+| What you will learn               | How to benchmark the memory bandwidth using STREAM.
+| Time to complete                  | 5 minutes
+
```

Comment:
As this is for Gen12 graphics, we  should include info on Dev Cloud use and a specific requirements section, not just a "optimized for" section

Reply:
Please provide some feedback on this
@moushumi-maria